state-chain/custom-rpc/src/lib.rs

```diff
@@ -448,6 +448,7 @@ pub struct SwappingEnvironment {
 	swap_retry_delay_blocks: u32,
 	max_swap_retry_duration_blocks: u32,
 	max_swap_request_duration_blocks: u32,
+	minimum_chunk_size: any::AssetMap<NumberOrHex>,
```

Comment:
See my other comment, `u32` is probably enough, and then we don't need NumberOrHex.